addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -617,6 +639,9 @@ const Wysiwyg = Widget.extend({
      * @override
      */
     destroy: function () {
+        // Redefine the getRangeAt function back to the native one.
+        Selection.prototype.getRangeAt = originalGetRangeAt;
```

Comment:
As validated by DMO, make it a general override instead of one only done for the duration of the edit mode. Unpatching that way is also unsafe as could unpatch other patches made by custom code by mistake.